### DISCUSSION TITLE
How to turn off virtual text showing keybindings for ask and edit?

### DISCUSSION THREAD

**Discussion by srgsanky** [yetone/avante.nvim#680] [Q&A] [answered]
How to turn off virtual text in visual mode that shows the keybindings to search with avante?

```
some_text  [<leader>aa: ask, <leader>ae: edit]
```

I couldn't find a relevant configuration https://github.com/yetone/avante.nvim/blob/main/lua/avante/config.lua

The logic that shows the virtual text https://github.com/yetone/avante.nvim/blob/0705234991d03170a72582085dc508600a03a779/lua/avante/selection.lua#L505-L516

I also submitted a feature request https://github.com/yetone/avante.nvim/issues/682

**Comment by romac** [comment, marked as answer]
Try setting `hints = { enabled = false }`, it worked for me.
